docs/user-guide/src/troubleshooting.md

```diff
@@ -88,3 +88,151 @@ of a forced deletion of its BareMetalHost object. If valid BMC credentials were
 provided, Ironic will keep checking the power state of the host and enforcing
 the last requested power state. The only solution is again to delete the
 Ironic's internal database.
+
+## BMH registration errors
+
+BMC credentials could be wrong or missing. These issues show up in the
+BareMetalHost’s status and as Events.
+
+Check both `kubectl describe bmh <name>` and recent Events for details.
+
+Example output:
+
+```text
+Normal  RegistrationError  23s   metal3-baremetal-controller  Failed to get 
+power state for node 67ac51af-a6b3. Error: Redfish exception occurred. 
+Error: HTTP GET https://192.168.111.1:8000/redfish/v1/Systems/... returned code 401.
+```
+
+## BMH inspection errors
+
+### The host is not able to communicate back results to Ironic
+
+If the host cannot communicate with Ironic, it will result in a timeout. Access
+to serial logs is needed to determine the exact issue.
```

Comment:
```suggestion
If the host cannot communicate with Ironic, it will result in a timeout.
Accessing serial logs is necessary to determine the exact issue.
```